examples/pylint/E0213_no_self_argument.py

```diff
@@ -0,0 +1,9 @@
+class MyClass:
```

Comment:
As a note (even though its in the commit), this was one that I forgot to change in the conversion (pylint considers something like _def method(asd)_ to be an error, _asd_ should be _self_, even though its not a Python error)